quickstart/src/index.js

```diff
@@ -83,7 +83,8 @@ $.getJSON('/token', function(data) {
 
 // Successfully connected!
 function roomJoined(room) {
-  activeRoom = room;
+  window.room = room;
+  window.activeRoom = activeRoom = room;
```

Comment:
`activeRoom` is set to `null` on disconnect, but I leave `room` set even after disconnect (in case one might want to check the final states of Participants, etc.). So I find it useful. WDYT?